### DISCUSSION TITLE
Are Operator policies supported by the RMQ topology operator?

### DISCUSSION THREAD

**Discussion by eisendle** [rabbitmq/messaging-topology-operator#709] [Q&A] [answered]
Hi,

I'm trying to create a operator policy via the topology operator in my openshift project, but there's only a Policy resource which only creates user policies.
Does the operator support operator policies?

**Comment by Zerpet** [comment, marked as answer]
Hi there,

It does not support operator policies at the moment. We had this issue, that closed due to staleness https://github.com/rabbitmq/messaging-topology-operator/issues/202

I've re-opened the issue and I will try to allocate some time to implement this feature, likely after Christmas. You are also welcome to submit a PR 🙂

**Comment by ansd** [comment]
@eisendle as promised, that feature is now implemented via https://github.com/rabbitmq/messaging-topology-operator/pull/752
